editions/test/tiddlers/tests/test-deserializers.js

```diff
@@ -0,0 +1,39 @@
+/*\
+title: test-deserializers.js
+type: application/javascript
+tags: [[$:/tags/test-spec]]
+
+Tests various core deserializers
+
+\*/
+(function(){
+
+	/* jslint node: true, browser: true */
+	/* eslint-env node, browser, jasmine */
+	/* eslint no-mixed-spaces-and-tabs: ["error", "smart-tabs"]*/
+	/* global $tw, require */
+	"use strict";
+	
+	describe("deserializer tests", function() {
+
+		function executeTestCase(title,expectedOutput) {
+			it("test case " + title, function() {
+				var tiddler = $tw.wiki.getTiddler(title);
+				expect($tw.wiki.deserializeTiddlers(tiddler.fields.type,tiddler.fields.text)).toEqual(expectedOutput);
+			});
+		}
+	
+		executeTestCase("dezerializer test data case 1",[ { text: '<!doctype html>\n', type: 'text/html' } ]);
+
+		executeTestCase("dezerializer test data case 2",[ { text: '<!doctype html>\n<html lang="en">\n<head>\n\t<meta charset="utf-8">\n\t<title>Test Data</title>\n</head>\n<body>\n</body>\n</html>\n', type: 'text/html' } ]);
```

Comment:
@Jermolene this and the above test are failing for me on the Windows 10 machine I am working on while away from home. My first guess is that it might be due to newline encoding?

@pmario could you perhaps confirm if you have the same problem? If it is something specific to my environment then it can be safely ignored.

![image](https://user-images.githubusercontent.com/68092/125780851-54e9eaa8-658d-4fc1-8588-05ce96e73588.png)

Reply:
> @Jermolene this and the above test are failing for me on the Windows 10 machine I am working on while away from home. My first guess is that it might be due to newline encoding?

That does sound very likely, might you have Git configured to convert LF to CRLF when it pulls?

Reply:
@Jermolene that is probably it, thank you. I am so used to a *nix based dev environment I hadn't even realized that git converting line endings was a thing! The travails of working in an unfamiliar OS.